Makefile

```diff
@@ -408,4 +410,13 @@ help: ## show this help screen
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 	@echo ''
 
+funcgen: lang/funcs/core/generated_funcs_tests.go lang/funcs/core/generated_funcs.go
```

Comment:
Is this the correct Makefile magic? (With the duplicate `funcgen` target?)
I was reading https://www.gnu.org/software/automake/manual/html_node/Multiple-Outputs.html
Does makefile allow the *.go in the input target?

Reply:
fixed